app/lib/utils/btc.js

```diff
@@ -93,6 +140,11 @@ export function fiatToSatoshis(fiat, price) {
   return btcToFiat(satoshisToBtc(satoshis), price)
 }
 
+export function fiatToMilliSatoshis(fiat, price) {
+  return btcToMillisatoshis(fiatToBtc(fiat, price))
```

Comment:
Looks like a mistake. Second return is unreachable

Reply:
Thanks. Have refactored.